src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import SpacerView from '@components/SpacerView';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWaitForNavigation from '@hooks/useWaitForNavigation';
+import * as Connections from '@libs/actions/connections';
+import Navigation from '@libs/Navigation/Navigation';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import FeatureEnabledAccessOrNotFoundWrapper from '@pages/workspace/FeatureEnabledAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import type {ToggleSettingOptionRowProps} from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function QuickbooksAdvancedPage({policy}: WithPolicyProps) {
+    const styles = useThemeStyles();
+    const waitForNavigate = useWaitForNavigation();
+    const {translate} = useLocalize();
+
+    const policyID = policy?.id ?? '';
+    const {autoSync, syncPeople, autoCreateVendor, pendingFields} = policy?.connections?.quickbooksOnline?.config ?? {};
+    const {bankAccounts, creditCards, otherCurrentAssetAccounts} = policy?.connections?.quickbooksOnline?.data ?? {};
+    const accountOptions = [...(bankAccounts ?? []), ...(creditCards ?? [])];
+    const invoiceAccountOptions = [...(bankAccounts ?? []), ...(otherCurrentAssetAccounts ?? [])];
+
+    const qboToggleSettingItems: ToggleSettingOptionRowProps[] = [
+        {
+            title: translate('workspace.qbo.advancedConfig.autoSync'),
+            subtitle: translate('workspace.qbo.advancedConfig.autoSyncDescription'),
+            isActive: Boolean(autoSync?.enabled),
+            onToggle: () =>
+                Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.AUTO_SYNC, {
+                    jobID: autoSync?.jobID ?? '',
+                    enabled: !autoSync?.enabled,
+                }),
```

Comment:
```suggestion
                Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.AUTO_SYNC, {
                    enabled: !autoSync?.enabled,
                }),
```

We only need to update the `enabled` field. This requires the following change:
1. In UpdatePolicyConnectionConfigParams (API type): make connectionName, settingName and settingValue props strings and remove the generic type
2. In updatePolicyConnectionConfig make settingValue type Partial<...>
3. In updatePolicyConnectionConfig use JSON.stringify on settingValue if it's an object